testutil/vault.go

```diff
@@ -140,10 +140,13 @@ func NewTestVault(t testing.T) *TestVault {
 	return NewTestVaultFromPath(t, "vault")
 }
 
-// NewTestVaultDelayed returns a test Vault server that has not been started.
-// Start must be called and it is the callers responsibility to deal with any
-// port conflicts that may occur and retry accordingly.
-func NewTestVaultDelayed(t testing.T) *TestVault {
+func NewTestVaultDelayedFromPath(t testing.T, binary string) *TestVault {
+	t.Helper()
+
+	if _, err := exec.LookPath(binary); err != nil {
+		t.Skipf("Skipping test %s, Vault binary not %q found in path.", t.Name(), binary)
```

Comment:
pretty sure `go test` already outputs the test name when you skip it (i think this was just copied from NewTestVaultFromPath anyway)

Reply:
I did copy this.  I'm happy to update both `Skipf` lines if you want.

Reply:
It's up to you. For nitpicky stuff like this we leave it up to the PR author. (it can be annoying to wait for a whole CI cycle for unimportant stuff)

Reply:
I removed the names of the tests to clean up with output a bit.